test/web-auth/hosted-pages.test.js

```diff
@@ -386,4 +386,67 @@ describe('auth0.WebAuth._universalLogin', function() {
       );
     });
   });
+
+  context('getSSOData', function() {
+    before(function() {
+      this.auth0 = new WebAuth({
+        domain: 'me.auth0.com',
+        clientID: '...',
+        redirectUri: 'http://page.com/callback',
+        responseType: 'code',
+        _sendTelemetry: false
+      });
+    });
+
+    afterEach(function() {
+      request.get.restore();
+    });
+
+    it('should call ssodata with all the options', function(done) {
+      stub(request, 'get', function(url) {
+        expect(url).to.be('https://me.auth0.com/user/ssodata/');
+        return new RequestMock({
+          headers: {},
+          cb: function(cb) {
+            cb(null, {
+              body: {
+                sso: false
```

Comment:
Can you please add one more parameter to this body? Same for the test below. Why? It makes some noise to me that the `sso: false` is also passed on many of the tests above of this PR, and I want to fully understand what this `getSSOData` call ends doing.

Reply:
sso:false is simulating the server response.

Reply:
Then why is it called "with all the options" if you're just passing 1? The previous tests I mentioned have more parameters. Is this correct with a single one?

Reply:
yeah.. I just copied/pasted from v8. I renamed the test